packages/blinks-core/src/hooks/useAction.ts

```diff
@@ -81,11 +81,23 @@ export function useAction({
     return () => {
       ignore = true;
     };
-  }, [actionApiUrl, isRegistryLoaded, supportStrategy]);
+  }, [actionApiUrl, isRegistryLoaded]);
```

Comment:
Please add eslint-ignore comment gere as well (since adapter and supportStrategy are still used inside the effect)